packages/maskbook/src/plugins/ITO/UI/StyledLinearProgress.tsx

```diff
@@ -1,13 +1,26 @@
-import { withStyles, LinearProgress } from '@material-ui/core'
+import { LinearProgress, LinearProgressProps } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
 
-export const StyledLinearProgress = withStyles({
-    root: {
-        height: 8,
-        borderRadius: 5,
-        background: 'rgba(255, 255, 255, 0.3)',
-    },
-    bar: {
-        borderRadius: 5,
-        backgroundColor: '#fff',
-    },
-})(LinearProgress)
+interface StyledLinearProgress extends LinearProgressProps {
+    barcolor?: string
```

Comment:
`color`

Reply:
Warning: Failed prop type: Invalid prop `color` of value `rgba(44, 164, 239)` supplied to `ForwardRef(LinearProgress)`, expected one of ["primary","secondary"].

Reply:
Well let me change this